templates/nodemon.json

```diff
@@ -0,0 +1,13 @@
+{
+    "restartable": "rs",
+    "ignore": [
+      ".git",
+      "node_modules/**/node_modules",
+      "tests/",
+      "static/"
+    ],
+    "env": {
+      "NODE_ENV": "development"
```

Comment:
I'd kind of prefer not to set any additional env vars in the setup. (Running boltzmann with `NODE_ENV=development` should be the same as running it without `NODE_ENV`, but by not providing the env var here, we encourage folks to avoid checking `process.env.NODE_ENV === 'development'`)